src/wp-admin/edit-form-blocks.php

```diff
@@ -140,15 +141,17 @@ function( $classes ) {
 			$response = wp_remote_get( $style );
 			if ( ! is_wp_error( $response ) ) {
 				$styles[] = array(
-					'css' => wp_remote_retrieve_body( $response ),
+					'css'            => wp_remote_retrieve_body( $response ),
+					'__unstableType' => 'theme',
 				);
 			}
 		} else {
 			$file = get_theme_file_path( $style );
 			if ( is_file( $file ) ) {
 				$styles[] = array(
-					'css'     => file_get_contents( $file ),
-					'baseURL' => get_theme_file_uri( $style ),
+					'css'            => file_get_contents( $file ),
+					'baseURL'        => get_theme_file_uri( $style ),
+					'__unstableType' => 'theme',
```

Comment:
Previously when working on presets we used the concept of origins to differentiate presets coming from core vs theme vs user. Styles are all the same type/structure they just have different sources/origins do you think it would make sense to name the property __unstableOrigin.

Reply:
See comment above https://github.com/WordPress/wordpress-develop/pull/1326#discussion_r645413564 I'd rather keep this as type.